### DISCUSSION TITLE
get total stock quantity in the footer and other things!

### DISCUSSION THREAD

**Discussion by anonkwe** [inventree/InvenTree#4523] [Q&A] [answered]
As @SchrodingersGat told me to, i open a new thread about this.

Here:
![image](https://user-images.githubusercontent.com/61838766/227412048-28f5c926-b0dc-43b3-b72d-5575222345e2.png)

also , in label printings, i would like to include packaging on my labels, but it does not seem to take any tag i try what would be the proper tag or code to have it  on my stock labels? example: {{ item.packaging }} or {{ stockitem.packaging }}. Is it also possible to get the part's supplier of the items that are included in the stock ? 

is there any documentation available where are listed every single possible tags or codes in label or report printing?

 please let me know thank you very much in advance!

**Comment by SchrodingersGat** [comment, marked as answer]
@anonkwe regarding the "total stock" request - I have raised a new [issue here](https://github.com/inventree/InvenTree/issues/4525)

> in label printings, i would like to include packaging on my labels, but it does not seem to take any tag i try what would be the proper tag or code to have it on my stock labels? example: {{ item.packaging }} or {{ stockitem.packaging }}. Is it also possible to get the part's supplier of the items that are included in the stock?

`{{ item.packaging }}` should work!

More docs available here:

https://docs.inventree.org/en/latest/report/context_variables/#stockitem
